avm/src/lib.rs

```diff
@@ -174,7 +179,7 @@ pub fn list_versions() -> Result<()> {
         if installed_versions.contains(v) {
             flags.push("installed");
         }
-        if current_version().unwrap() == v.clone() {
+        if current_version().is_ok() && current_version().unwrap() == v.clone() {
```

Comment:
```suggestion
        if matches!(current_version(), Ok(cv) if cv == *v) {
```
Version without `unwrap` :slightly_smiling_face:

Reply:
very nice! :dancers: